web/data/affiliated_pkgs.json

```diff
@@ -0,0 +1,48 @@
+{
+  "plasmapy": {
+    "maintainer": {
+      "name": null,
+      "contact": null
+    },
+    "urls": {
+      "home": "https://www.plasmapy.org",
+      "repo": "https://github.com/plasmapy/plasmapy",
+      "image": "/images/plasmapy-graphic-circular.png"
+    },
+    "description": {
+      "short": "A open-source Python 3 package for plasma physics.",
+      "long": "PlasmaPy is an open-source community developed Python 3 package for plasma physics.  PlasmaPy intends to be a collection of functionality commonly used and shared between plasma physicists and researchers globally."
+    },
+    "affiliated": true,
+    "keywords": ["plasma", "physics", "python 3"],
+    "dates": {
+      "added": "2020 November 12",
+      "last_updated": "2020 November 12"
+    },
+    "status": "perpetual beta",
+    "license": "BSD 3-clause with patent grant"
+  },
+  "tofu": {
```

Comment:
I need to finally catch up on the tofu stuff... I just know it goes well with chills.

Reply:
Yeah, next week is going to catch up on everything dripped over the last two weeks.